src/app/globals.css

```
@@ -3,120 +3,232 @@
+  }
+}
+
+/* Animations */
+@theme inline {
+  --animate-first: moveVertical 30s ease infinite;
+  --animate-second: moveInCircle 20s reverse infinite;
+  --animate-third: moveInCircle 40s linear infinite;
+  --animate-fourth: moveHorizontal 40s ease infinite;
+  --animate-fifth: moveInCircle 20s ease infinite;
+
+  @keyframes moveHorizontal {
```

Stylelint (17.6.0)

[error] Expected keyframe name "moveHorizontal" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)